packaging/docker/samples/python/app/server.py

```diff
@@ -23,10 +23,13 @@
 
 app = Flask(__name__)
 
-fdb.api_version(int(os.getenv('FDB_API_VERSION')))
+fdb.api_version(int(os.getenv("FDB_API_VERSION")))
+fdb.options.set_knob("disable_posix_kernel_aio=1")
```

Comment:
Are you sure this needs to be set in the client? I would expect that it only needs to be set on the fdbserver processes. We set it on fdbserver in our local test configs for the Kubernetes operator, and things seem to work fine without setting it on clients or CLIs.

Reply:
My limited experience indicates that setting the knob in the client was necessary. I don't have sufficient understanding of the internals to defend that experience.